packages/lib/src/core/Analytics/types.ts

```diff
@@ -87,41 +107,50 @@ export interface AnalyticsObject {
     configData?: Record<string, string | boolean>;
 }
 
-export type AnalyticsEvent = (typeof ANALYTICS_EVENT)[keyof typeof ANALYTICS_EVENT];
-
-export type CreateAnalyticsObject = Omit<AnalyticsObject, 'timestamp' | 'id'> & { event: AnalyticsEvent };
+export type NewAnalyticsEventObjectInfo = {
+    category: AnalyticsEvent;
+    type?: string;
+    target?: string;
+    issuer?: string;
+    isExpress?: boolean;
+    expressPage?: string;
+    isStoredPaymentMethod?: boolean;
+    brand?: string;
+    validationErrorCode?: string;
+    validationErrorMessage?: string;
+    configData?: Record<string, string | boolean>;
+    component?: string;
+};
 
-export type AnalyticsInitialEvent = {
-    containerWidth: number;
-    component: string;
-    flavor: string;
-    paymentMethods?: any[];
-    sessionId?: string;
+export type NewAnalyticsEventObjectLog = {
+    category: AnalyticsEvent;
+    type?: string;
+    message?: string;
+    subType?: string;
+    result?: string;
+    component?: string;
+    target?: string; // is this ever used?
 };
 
-export type AnalyticsConfig = {
-    analyticsContext?: string;
-    clientKey?: string;
-    locale?: string;
-    amount?: PaymentAmount;
-    loadingContext?: string;
+export type NewAnalyticsEventObjectError = {
```

Comment:
I see there are couple of types prefixed by `New` -  what is the intention of using the `New`?  Are they just normal types and you are using `New` because you are refactoring/renaming things?

In general, `New` is a special keyword to create objects and it is not used to define types. It seems a bit off to me

Reply:
It was just due to the refactoring - to easily distinguish the "new" stuff I was creating from what was already there.
I'll replace it all once the final shape of the code is decided